bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,259 @@ class CoffeeLintBear:
+            ```
+            1:2, 3:4
+            ``` is prohibited whereas
+            ```
+            {1:2, 3:4}
+            ``` is accepted.
+        :param: no_function_call_without_parentheses:
+            prohibits implicit parentheses on function calls.
+        :param: no_interpolation_in_single_quotes:
+            Prohibits string interpolation in a single quoted string.
+
+            Example: If ``no_interpolation_in_single_quotes = True`` then
+            ```
+            f = '#{bar}'
+            ``` is prohibited whereas
+            ```
+            f = "#{bar}"
+            ``` is accepted.
+        :param: no_stand_alone_at:
+            Checks if stand alone at ``@`` are in use.
+        :param: no_throwing_strings:
+           Prohibits throwing string literals or interpolation.
+        :param: no_trailing_semicolons:
+           Prohibits trailing semicolons since they are not useful.
+
+           Example: If ``no_trailing_semicolon = True``
+           ```
+           x = '1234'; console.log(x)
+           ``` here the semicolon is meaningfull.
+           ```
+           alert('end of line');
+           ``` the semicolon is useless.
+        :param: no_trailing_whitespaces:
+            Checks if they are trailing whitespace in the code.
+        :param: apply_no_trailing_whitespaces_in_comments:
+            Determines if ``no_trailing_whitespace`` rule should be applied to
+            comments.
+        :param: apply_no_trailing_whitespaces_in_empty_line:
+            Determines if ``no_trailing_whitespace`` rule should be applied to
+            empty lines.
+        :param: no_unecessary_double_quotes:
+            Prohibits double quotes unless string interpolation is used or the
+            string contains single quotes.
+        :param: use_english_operator:
+            Determines if ``and``, ``or``, ``is`` and ``isnt`` should be use
+            instead of ``&&``, ``||``, ``==`` and ``!=``.
+        :param: space_around_operators:
+            Enforces that operators have spaces around them.
+        :param: space_after_comma:
+            Checks if there is a space after commas.
+        :param: cyclomatic_complexity:
+            Determines if the complexity of the file should be examine.
+        :param: cyclomatic_complexity_value:
+            Cyclomatic complexity of the file.
+        :param: duplicate_keys:
+            Prevents defining duplicate keys in object literals and classes.
+        :param: enforce_parentheses_on_empty_constructor:
+            Requires constructors with no parameters to include parentheses.
+        :param: line_endings:
+            Ensures that the file uses only ``windows`` or ``unix`` line
+            endings.
+        :param: line_endings_option:
+            The option to ``line_endings``, its value is either ``unix`` or
+            ``windows``.
+        :param: no_this:
+            prohibits ``this``. Use ``@`` instead.
+        :param: no_plusplus:
```

Comment:
ah i hate this name, the decrementation is minusminus so ... please find a better one
maybe ``no_decr_or_incrementation_operators``